Reword the watcher 'always' and 'never' condition docs (#86105)
elastic/elasticsearch: x-pack/docs/en/watcher/condition.asciidoc

++++

When a watch is triggered, its condition determines whether or not to execute
the watch actions. {watcher} supports the following condition types:

* <<condition-always,`always`>>: a condition whose result is `true`, so the watch
actions are always executed.

> * <<condition-never,`never`>>: a condition whose result is `false`, so the watch
> actions are never executed.

Reviewer: ```suggestion
* <<condition-never,`never`>>: The condition always evaluates to `false`, so the watch
actions are never executed.
```
